modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava

```diff
@@ -57,6 +57,39 @@ public class UpgradeCatchAllCheck {
 		_commerceAccountHelper.getCurrentCommerceAccount(null);
 	}
 
+	public void method(
+		long groupId, boolean privateLayout, long layoutId, long parentLayoutId,
+		Map<java.util.Locale, String> nameMap,
+		Map<java.util.Locale, String> titleMap,
+		Map<java.util.Locale, String> descriptionMap,
+		Map<java.util.Locale, String> keywordsMap,
+		Map<java.util.Locale, String> robotsMap, String type, boolean hidden,
+		Map<java.util.Locale, String> friendlyURLMap, boolean hasIconImage,
+		byte[] iconBytes, ServiceContext serviceContext){
+
+		LayoutLocalServiceUtil.updateLayout(groupId, privateLayout, layoutId, parentLayoutId, nameMap, titleMap, descriptionMap, keywordsMap, robotsMap, type, hidden, friendlyURLMap, hasIconImage, iconBytes, 0, 0, 0, serviceContext);
+		LayoutServiceUtil.updateLayout(groupId, "test123", 123, parentLayoutId, nameMap, titleMap, descriptionMap, keywordsMap, robotsMap, test, hidden, friendlyURLMap, hasIconImage, iconBytes, serviceContext);
+		LayoutServiceUtil.updateLayout(groupId, privateLayout, layoutId, parentLayoutId, nameMap, titleMap, descriptionMap, keywordsMap, robotsMap, type, hidden, friendlyURLMap, hasIconImage, iconBytes, 0, 0, 0, serviceContext);
+		_layoutLocalService.updateLayout(groupId, privateLayout, layoutId, parentLayoutId, nameMap, titleMap, descriptionMap, keywordsMap, robotsMap, type, hidden, friendlyURLMap, hasIconImage, iconBytes, 0, 0, 0, serviceContext);
+		_layoutService.updateLayout(groupId, privateLayout, layoutId, parentLayoutId, nameMap, titleMap, descriptionMap, keywordsMap, robotsMap, type, hidden, friendlyURLMap, hasIconImage, iconBytes, 0, 0, 0, serviceContext);
+	}
+
+	public void method(
+		long userId, long groupId, boolean privateLayout, long parentLayoutId,
+		long classNameId, long classPK, Map<java.util.Locale, String> nameMap,
+		Map<java.util.Locale, String> titleMap,
+		Map<java.util.Locale, String> descriptionMap,
+		Map<java.util.Locale, String> keywordsMap,
+		Map<java.util.Locale, String> robotsMap, String type,
+		String typeSettings, boolean hidden, boolean system,
+		Map<java.util.Locale, String> friendlyURLMap,
+		ServiceContext serviceContext) {
+
+		LayoutLocalServiceUtil.addLayout(userId, groupId, privateLayout, parentLayoutId, classNameId, classPK, nameMap, titleMap, descriptionMap, keywordsMap, robotsMap, type, typeSettings, hidden, system, friendlyURLMap, 0, serviceContext);
+		_layoutLocalService.addLayout(userId, "test123", 123, parentLayoutId, classNameId, classPK, nameMap, titleMap, descriptionMap, keywordsMap, robotsMap, type, typeSettings, hidden, system, friendlyURLMap, serviceContext);
+		_layoutLocalService.addLayout(userId, groupId, privateLayout, parentLayoutId, classNameId, classPK, nameMap, titleMap, descriptionMap, keywordsMap, robotsMap, type, typeSettings, hidden, system, friendlyURLMap, 0, serviceContext);
+	}
```

Comment:
@MicaelleSilvaa please run the SF again on these test files to ensure that the order is correct.

Reply:
resolved